drivers/overlay/overlay.go

```diff
@@ -2554,18 +2568,11 @@ func (d *Driver) getAdditionalLayerPath(tocDigest digest.Digest, ref string) (st
 			ref = refElem
 		}
 		target := path.Join(ls.path, ref, tocDigest.String())
-		// Check if all necessary files exist
-		for _, p := range []string{
-			filepath.Join(target, "diff"),
-			filepath.Join(target, "info"),
-			filepath.Join(target, "blob"),
-		} {
-			if err := fileutils.Exists(p); err != nil {
-				wrapped := fmt.Errorf("failed to stat additional layer %q: %w", p, err)
-				return "", fmt.Errorf("%v: %w", wrapped, graphdriver.ErrLayerUnknown)
-			}
+		err := checkIfAllNesseseryFilesExistsInLayerStore(target)
+		if err == nil {
+			return target, nil
 		}
-		return target, nil
+		logrus.Warn(err)
```

Comment:
_If_ there are going to be multiple ALS stores, it’s nothing unexpected if a layer does not exist in the first one, but exits in the second one. So this should not be a warning — at most a debug log.